src/swapd/runtime.rs

```diff
@@ -1434,8 +1429,8 @@ impl Runtime {
                         }
                     }
                     Event::TransactionConfirmations(TransactionConfirmations {
-                        id,
-                        block,
+                        id: _,
```

Comment:
the above examples, but these ones on ignoring stuff, i dont like so much

Reply:
I find it cleaner like this:
``` rust
                  Event::TransactionConfirmations(TransactionConfirmations {
                        block,
                        ..
                   }
```

Reply:
I added this, because I thought you are destructuring everything to make  it easier for yourself later on. Changing this to `..` instead then.